src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -12,8 +12,17 @@
                 i18n>Grafana Dashboard doesn't exist. Please refer to
   <cd-doc section="grafana"></cd-doc> on how to add dashboards to Grafana.</cd-alert-panel>
 
+  <cd-alert-panel type="info"
+                  *ngIf="!iframeLoaded"
+                  i18n>The embedded Grafana Dashboard was blocked due to a certificate error, please click <a [href]="baseUrl"
```

Comment:
Instead of `baseUrl` we can provide the `grafanaSrc` which is actually the coorect url for the datasource.
```suggestion
                  i18n>The embedded Grafana Dashboard was blocked due to a certificate error, please click <a [href]="grafanaSrc"
```